mayastor/src/bdev/nexus/nexus_child_error_store.rs

```diff
@@ -105,61 +113,138 @@ impl NexusErrStore {
         self.next_record_index =
              // consider using a mask if we restrict the size to a power-of-2
             (self.next_record_index + 1) % self.records.len();
-        // debug!("added record - buffer is {}", &self);
     }
 
-    pub fn query(
+    #[inline]
+    fn is_wanted_record(
```

Comment:
would "find" be better? or maybe, has_error(..) ?

Reply:
Ok, I'll go with has_error().